docs/reference/files/tests-unit-test-charm-py-file.rst

```diff
@@ -0,0 +1,41 @@
+.. _tests-unit-test-charm-py-file:
+
+
+``tests/unit/test_charm.py`` file
+=================================
+
+    See also: `How to write unit tests for a charm <https://juju.is/docs/sdk/write-a-unit-test-for-a-charm>`_
+
+The ``tests/unit/test_charm.py`` file is the companion to
```

Comment:
Same here - not all profiles generate this file. I think specifically the 12-factor ones don't.